main/lsp/DefLocSaver.cc

```diff
@@ -53,10 +53,13 @@ ast::ExpressionPtr DefLocSaver::postTransformUnresolvedIdent(core::Context ctx,
     auto &id = ast::cast_tree_nonnull<ast::UnresolvedIdent>(tree);
     if (id.kind == ast::UnresolvedIdent::Kind::Instance || id.kind == ast::UnresolvedIdent::Kind::Class) {
         core::ClassOrModuleRef klass;
-        // Logic cargo culted from `global2Local` in `walker_build.cc`.
```

Comment:
This comment seems to be not true anymore

Reply:
https://github.com/sorbet/sorbet/blob/5bdbbe9e/cfg/builder/builder_walk.cc#L65

Looks like the code is still there but renamed. 🤔 

How does that code not crash?

Reply:
Oh, I think I know. I think this code runs before a fake `static-init` method is created to contain this AST node, so it is owned by the class.

Good catch!